tests/integration/test_data_cli.py

```diff
@@ -218,6 +219,100 @@ def test_search_create_daily_email(invoke, search_result):
     assert json.loads(result.output) == search_result
 
 
+@respx.mock
+@pytest.mark.asyncio
+@pytest.mark.parametrize("filter", ['{1:1}', '{"foo"}'])
+@pytest.mark.parametrize(
```

Comment:
how does changing item_types apply to what is being tested here?

Reply:
you're right, it doesn't! I had just included all of the options just to test them but that's already being tested elsewhere so we don't need it.